src/server/handlers/github.js

```diff
@@ -76,16 +75,13 @@ export const newIntegration = (req, res) => {
   });
 };
 
-const getRequest = (account, repo) => {
+const getRequest = (account, repo, token) => {
   return {
     url: GITHUB_API_ENDPOINT + `/repos/${account}/${repo}/hooks`,
```

Comment:
why string concat and template literal? couldn't this just be `${GITHUB_API_ENDPOINT}/repos/...`